openslides_backend/action/motion_state/create_update_delete.py

```diff
@@ -41,6 +41,21 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         return instance
 
 
+class MotionStateDelete(DeleteAction):
+    """
+    Action to delete a motion state.
+    """
+
+    def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
+        motion_state = self.database.get(
+            FullQualifiedId(self.model.collection, instance["id"]),
+            ["first_state_of_workflow_id"],
+        )
+        if motion_state.get("first_state_of_workflow_id"):
+            raise ActionException("You cannot delete the first state of the workflow.")
+        return instance
```

Comment:
This can be easier implemented by setting `on_delete=PROTECT` on the `first_state_of_workflow_id` field in the models.yml. Please remove this and update the models accordingly for cleaner code